src/vmm_core.ml

```diff
@@ -241,21 +241,23 @@ module Policy = struct
     let bridges =
       if String_set.is_empty res.bridges then "" else ", bridges: "
     in
-    Fmt.pf ppf "policy: %d vms %a cpus %d MB memory %a block%s%a"
-      res.vms pp_is res.cpuids res.memory
+    Fmt.pf ppf "policy: %d unikernels %a cpus %d MB memory %a block%s%a"
+      res.unikernels pp_is res.cpuids res.memory
       Fmt.(option ~none:(any "no") (int ++ any " MB")) res.block
       bridges
       Fmt.(list ~sep:(any ", ") string) (String_set.elements res.bridges)
 
-  let usable { vms ; cpuids ; memory ; _ } =
-    if vms <= 0 then
-      Error (`Msg "Unusable policy with no VMs")
+  let usable { unikernels ; cpuids ; memory ; block ; _ } =
+    if unikernels <= 0 then
+      Error (`Msg "Unusable policy with no unikernels")
     else if IS.is_empty cpuids then
       Error (`Msg "Unusable policy with no CPUids")
     else if memory <= 16 then
-      Error (`Msg "Unusable policy with memory <= 16 MB")
-    else
-      Ok ()
+      Error (`Msg ("Unusable policy with memory " ^ string_of_int memory ^ " MB <= 16 MB"))
+    else match block with
+      | None -> Ok ()
+      | Some x when x >= 0 -> Ok ()
+      | Some x -> Error (`Msg ("Unusable policy with block " ^ string_of_int x ^ " MB < 0 MB"))
```

Comment:
I'm not sure it's exactly unusable as it would be equivalent to `None`. It's a redundant representation, though. I guess this could break policies made previously?!

Reply:
Ah, `Vmm_core.Policy.usable` is only ever called in the client before adding a new policy. So this should not break existing policies.

Reply:
Also, I'm not sure this is actually an unusable policy...

Reply:
It's a check that if block is specified, it is > 0. I do think this makes sense, or doesn't it?

Reply:
Ah sorry you are right. I somehow read it as x > 0, and and x = 0 makes sense (if redundant)